src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person,
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <Link
+      to={{
+        pathname: `../${person.slug}`,
+        search: searchParams.toString(),
```

Comment:
do we really need to add ` search: searchParams.toString(),` ?
try to test

Reply:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/101358500/f9bc6728-5ecc-4f8b-85e8-aa2494b71b9f)
it doesn't work, i should put string here